ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc

```diff
@@ -35,12 +35,28 @@
 #include <DlSystem/ITensorFactory.hpp>
 #include <DlSystem/TensorMap.hpp>
 
+#if defined(__ANDROID__)
+#include <jni.h>
+#endif
+
+/**
+ * @brief Macro for debug mode.
+ */
+#ifndef DBG
+#define DBG FALSE
+#endif
+
 namespace nnstreamer {
 namespace tensor_filter_snpe {
 
 extern "C" {
-  void _init_filter_snpe (void) __attribute__ ((constructor));
-  void _fini_filter_snpe (void) __attribute__ ((destructor));
+#if defined(__ANDROID__)
+  static gboolean _snpe_set_env (JNIEnv * env, jobject context);
```

Comment:
IMO, this line is unnecessary.

Reply:
Thanks! You are right. I removed the line.